cli/cmd/runtime/start.go

```diff
@@ -32,19 +32,20 @@ import (
 // Env var keys must be prefixed with RILL_RUNTIME_ and are converted from snake_case to CamelCase.
 // For example RILL_RUNTIME_HTTP_PORT is mapped to Config.HTTPPort.
 type Config struct {
-	HTTPPort             int           `default:"8080" split_words:"true"`
-	GRPCPort             int           `default:"9090" split_words:"true"`
-	LogLevel             zapcore.Level `default:"info" split_words:"true"`
-	MetastoreDriver      string        `default:"sqlite"`
-	MetastoreURL         string        `default:"file:rill?mode=memory&cache=shared" split_words:"true"`
-	AllowedOrigins       []string      `default:"*" split_words:"true"`
-	AuthEnable           bool          `default:"false" split_words:"true"`
-	AuthIssuerURL        string        `default:"" split_words:"true"`
-	AuthAudienceURL      string        `default:"" split_words:"true"`
-	SafeSourceRefresh    bool          `default:"false" split_words:"true"`
-	ConnectionCacheSize  int           `default:"100" split_words:"true"`
-	QueryCacheSize       int           `default:"10000" split_words:"true"`
-	AllowHostCredentials bool          `default:"false" split_words:"true"`
+	HTTPPort                     int           `default:"8080" split_words:"true"`
+	GRPCPort                     int           `default:"9090" split_words:"true"`
+	LogLevel                     zapcore.Level `default:"info" split_words:"true"`
+	MetastoreDriver              string        `default:"sqlite"`
+	MetastoreURL                 string        `default:"file:rill?mode=memory&cache=shared" split_words:"true"`
+	AllowedOrigins               []string      `default:"*" split_words:"true"`
+	AuthEnable                   bool          `default:"false" split_words:"true"`
+	AuthIssuerURL                string        `default:"" split_words:"true"`
+	AuthAudienceURL              string        `default:"" split_words:"true"`
+	SafeSourceRefresh            bool          `default:"false" split_words:"true"`
+	ConnectionCacheSize          int           `default:"100" split_words:"true"`
+	QueryCacheSize               int           `default:"10000" split_words:"true"`
+	AllowHostCredentials         bool          `default:"false" split_words:"true"`
+	DisablePathAccessOutsideRepo bool          `default:"true" split_words:"true"`
```

Comment:
Yeah I think its better to merge both variables to `AllowHostAccess ` since both essentially does the same. I also can't think of any situation where we would want to enable host credentials but disable path access outside repo and vice versa.
I will make the changes.